README.md

```diff
@@ -84,9 +84,22 @@ docker-compose up
 You can also run the docker container with ngrok for testing purposes by running.
 
 ```
-docker-compose -f docker-compose-ngrok.yml up
+docker-compose -f docker-compose.ngrok.yml up
```

Comment:
I think these instructions are outdated, no?  ngrok is the default when you run `./manage start` and to run with an nginx tunnel you run `./manage start --tunnel`?